proto/grp.proto

```diff
@@ -61,7 +72,60 @@ message GetGroupResponse {
   string owned_domain = 3;
 }
 
+message GetGroupUsersRequest {
+  context.RequestContext request_context = 1;
+
+  // The ID of the group to get the users from.
+  string group_id = 2;
+
+  // Membership status filter (optional).
+  GroupMembershipStatus group_membership_status = 3;
+}
+
+message GroupUser {
+  user_id.DisplayUser user = 1;
+  GroupMembershipStatus group_membership_status = 2;
+}
+
+message GetGroupUsersResponse {
+  context.ResponseContext response_context = 1;
+
+  // Users related to the group that match the request criteria.
+  repeated user_id.DisplayUser users = 2;
```

Comment:
This should be GroupUser, right?

Reply:
I had to grep for this but I think in our protos our convention is to stick to the singular form for field names.

tylerw@lunchbox:~/buildbuddy$ ack "repeated" proto/invocation.proto 
  repeated InvocationEvent event = 2;
  repeated string pattern = 8;
  repeated command_line.CommandLine structured_command_line = 12;
  repeated Invocation invocation = 2;
  repeated Invocation invocation = 2;
  repeated InvocationStat invocation_stat = 2;
tylerw@lunchbox:~/buildbuddy$ ack "repeated" proto/execution
execution.proto~        execution_stats.proto   execution_stats.proto~  
tylerw@lunchbox:~/buildbuddy$ ack "repeated" proto/execution_stats.proto
  repeated Execution execution = 2;
tylerw@lunchbox:~/buildbuddy$